Rover/sailboat.cpp

```diff
@@ -102,10 +103,26 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("LOIT_RADIUS", 9, Sailboat, loit_radius, 5),
 
+    // @Param: OPTIONS
+    // @DisplayName: Sailboat options
+    // @Description: Sailboat setup option bits only available for separate mainsail RC channel.
+    // @Bitmask: 0: TODO Mainsail works as traditional RC sail in manual. Range 1000 usec to 2000 usec. 1000 usec is fully sheeted in. 2000 usec is fully sheeted out
+    // @Bitmask: 1: reserved 1
+    // @Bitmask: 2: reserved 2
+    // @Bitmask: 3: reserved 3
+    // @Bitmask: 4: TODO mainsail channel modifies angle of attack in auto or acro. MainSailRCIn range {1000 to 1900} usec. Sail angle of attack = (1.0f - (MainsailChannelIn - 1000.0f)/900.0f) * SAIL_ANGLE_IDEAL. MainSailRCIn range {1900 +}  usec. Sail relaxed.
+    // @Bitmask: 5: reserved 4
+    // @Bitmask: 6: reserved 5
+    // @Bitmask: 7: Detect and get out of "in Irons" in AUTO mode
+     AP_GROUPINFO("OPTIONS", 10, Sailboat, options, 0),
+
     AP_GROUPEND
 };
 
-
+namespace {
+   uint32_t constexpr getOutOfInIronsInAutoBit = 7U;
+   uint32_t constexpr getOutOfInIronsInAutoMask = (1U << getOutOfInIronsInAutoBit);
+}
```

Comment:
enum class in the header. We also like to add little "option set" helpers that take the enum value were checking and return bool